### PR TITLE
preparation for Membership Framework

### PR DESCRIPTION
**Pull request by retif**
Updates in composer.json:
-  "php": ">=7.1",
- core patch "2996114 - JSON:API Argument error"

Implemented new 'Two Columns with left sidebar' layout for landing pages.

Wrapped field items in field templates with "field item" class to ease fields styling.

Updated theme file for Carnation to fix logo rendering logic.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: composer.json [hunk 1]
```diff
@@ -299,6 +300,7 @@
         "drupal/twig_tweak": "^2.4",
         "drupal/image_widget_crop": "^2.2",
         "drupal/geysir": "^1.1",
-        "drupal/media_directories": "^1.0"
+        "drupal/media_directories": "^1.0",
+        "openy/openy_memberships": "dev-master"
```
Comment (anpolimus) [inline composer.json]: Please delete it

**Review by podarok — APPROVED** [review]
Looks good
Let's have some steps for review @anpolimus before merging

**Comment by podarok** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2209/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2209/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2209/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2209/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2209/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2209/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2209/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2209/

**Pull request merged**